### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by kavithakpits** [akeneo/MagentoConnectorBundle#476]
successfully installed magento connector but after that akeneo website gone. Cache folder was cleared but not working.

**Comment by damien-carcel** [comment]
Hi.

Could you give me more details, please? What version of Akeneo are you using? You say your cache is not working, do you have any error messages? Did you follow the entire install procedure without errors at any step?

**Comment by kavithakpits** [comment]
Hi,
thanks for the reply. These are the steps that i followed

$ php composer.phar require akeneo/magento-connector-bundle:1.2.*
$bundles[] = new Pim\Bundle\MagentoConnectorBundle\PimMagentoConnectorBundle();
php app/console doctrine:schema:update --force
php app/console pim:installer:assets

But app_dev.php is working but magento connector is not showing in export profiles. could you please help me

**Comment by damien-carcel** [comment]
You should also clear the cache completely:
$ rm -rfv app/cache/*
$ php app/console cache:clear --env=prod
(and without the --env=prod option for dev environment).
Then restart your apache server. It is not mandatory, but in certain case it can make a difference.

Cheers,
Damien

**Comment by kavithakpits** [comment]
Thanks for the reply. I am not aware of symfony. iam getting this exception 

[Symfony\Component\DependencyInjection\Exception\ParameterNotFoundException]
  The service "actualys_drupal_commerce_connector.reader.doctrine.obsoleteproduct" has a dependency on a non-existent parameter "pim_base_
  connector.reader.doctrine.obsoleteproduct.class". Did you mean one of these: "pim_base_connector.reader.doctrine.bulk_product.class", "p
  im_base_connector.reader.orm.category.class", "pim_base_connector.reader.file.csv_product.class", "pim_base_connector.reader.file.csv_ca
  tegory.class", "pim_base_connector.processor.product.class", "pim_base_connector.writer.doctrine.product.class", "pim_base_connector.wri
  ter.file.product.class", "pim_base_connector.writer.file.csv_product.class", "pim_base_connector.reader.doctrine.product.class"?

**Comment by kavithakpits** [comment]
Hi,
Removing drupal connector, successfully cleared cache but after that i am not getting export files page

**Comment by damien-carcel** [comment]
I was going to say that you have a problem with this drupal connector.

You restart your apache server too? What version of the PIM are you using? Also, could you copy/past your AppKernel.php? Maybe you past the "$bundles[] = new Pim\Bundle\MagentoConnectorBundle\PimMagentoConnectorBundle();" in the wrong place, as it seems that Akeneo is not aware of the bundle.

**Comment by kavithakpits** [comment]
<?php

use Symfony\Component\Config\Loader\LoaderInterface;
use Oro\Bundle\DistributionBundle\OroKernel;

/**
 * PIM AppKernel
 *
 * @author    Nicolas Dupont <nicolas@akeneo.com>
 * @copyright 2013 Akeneo SAS (http://www.akeneo.com)
 * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
 */
class AppKernel extends OroKernel
{
	
	public function __construct($environment, $debug)
	{
		ini_set( 'date.timezone', 'Asia/Calcutta' );
		parent::__construct($environment, $debug);
	}
    /**
     * {@inheritdoc}
     */
    public function registerBundles()
    {
        $bundles = array();

        if (in_array($this->getEnvironment(), array('dev', 'test', 'behat'))) {
		
            $bundles[] = new Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
            $bundles[] = new Sensio\Bundle\DistributionBundle\SensioDistributionBundle();
            $bundles[] = new Sensio\Bundle\GeneratorBundle\SensioGeneratorBundle();
        }
		else{
			$bundles[] = new Pim\Bundle\MagentoConnectorBundle\PimMagentoConnectorBundle();
			$bundles[] = new Actualys\Bundle\DrupalCommerceConnectorBundle\ActualysDrupalCommerceConnectorBundle();
		}

        $oroBundles = parent::registerBundles();
        $exclude = array(
            'Oro\Bundle\ImportExportBundle\OroImportExportBundle',
            'Oro\Bundle\QueryDesignerBundle\OroQueryDesignerBundle',
            'JMS\JobQueueBundle\JMSJobQueueBundle',
            'Oro\Bundle\AddressBundle\OroAddressBundle',
            'Oro\Bundle\SoapBundle\OroSoapBundle',
            'Oro\Bundle\SyncBundle\OroSyncBundle',
            'Oro\Bundle\WorkflowBundle\OroWorkflowBundle',
            'Oro\Bundle\NotificationBundle\OroNotificationBundle',
            'Oro\Bundle\CalendarBundle\OroCalendarBundle',
            'Oro\Bundle\CronBundle\OroCronBundle',
            'Oro\Bundle\SearchBundle\OroSearchBundle',
            'Oro\Bundle\TagBundle\OroTagBundle'
        );
        foreach ($oroBundles as $bundleIndex => $bundle) {
            if (in_array(get_class($bundle), $exclude)) {
                unset($oroBundles[$bundleIndex]);
            }
        }
        $bundles = array_merge($oroBundles, $bundles);

        $pimDepBundles = array(
            // Uncomment the following line to use MongoDB implementation
            // new Doctrine\Bundle\MongoDBBundle\DoctrineMongoDBBundle(),
            new Knp\Bundle\GaufretteBundle\KnpGaufretteBundle(),
            new APY\JsFormValidationBundle\APYJsFormValidationBundle(),
            new Akeneo\Bundle\MeasureBundle\AkeneoMeasureBundle(),
            new Akeneo\Bundle\BatchBundle\AkeneoBatchBundle()
        );
        $bundles = array_merge($bundles, $pimDepBundles);

        $pimBundles = array(
            // BAP overriden bundles
            new Pim\Bundle\NavigationBundle\PimNavigationBundle(),
            new Pim\Bundle\FilterBundle\PimFilterBundle(),
            new Pim\Bundle\UserBundle\PimUserBundle(),
            new Pim\Bundle\JsFormValidationBundle\PimJsFormValidationBundle(),
            new Pim\Bundle\EntityBundle\PimEntityBundle(),

            // PIM bundles
            new Pim\Bundle\DashboardBundle\PimDashboardBundle(),
            new Pim\Bundle\InstallerBundle\PimInstallerBundle(),
            new Pim\Bundle\UIBundle\PimUIBundle(),
            new Pim\Bundle\CatalogBundle\PimCatalogBundle(),
            new Pim\Bundle\DataGridBundle\PimDataGridBundle(),
            new Pim\Bundle\TranslationBundle\PimTranslationBundle(),
            new Pim\Bundle\ImportExportBundle\PimImportExportBundle(),
            new Pim\Bundle\VersioningBundle\PimVersioningBundle(),
            new Pim\Bundle\WebServiceBundle\PimWebServiceBundle(),
            new Pim\Bundle\EnrichBundle\PimEnrichBundle(),
            new Pim\Bundle\BaseConnectorBundle\PimBaseConnectorBundle(),
            new Pim\Bundle\TransformBundle\PimTransformBundle()
        );

        $bundles = array_merge($bundles, $pimBundles);

        return $bundles;
    }

    /**
     * {@inheritdoc}
     */
    public function registerContainerConfiguration(LoaderInterface $loader)
    {
        $loader->load(__DIR__.'/config/config_'.$this->getEnvironment().'.yml');

        if (is_file($file = __DIR__.'/config/config_'.$this->getEnvironment().'_local.yml')) {
            $loader->load($file);
        }

        if (isset($this->bundleMap['DoctrineMongoDBBundle'])) {
            $loader->load(__DIR__ .'/config/config_mongodb.yml');
        }
    }
}

**Comment by kavithakpits** [comment]
Is this needed in composer.json
akeneo/connector-mapping-bundle:v1.0.0-BETA3

**Comment by damien-carcel** [comment]
No, this is not anymore, it was a dependency for MagentoConnectorBundle 1.1, it is now included in MagentoConnectorBundle 1.2.

However, I see in your AppKernel.php that you misplace the line

    "$bundles[] = new Pim\Bundle\MagentoConnectorBundle\PimMagentoConnectorBundle();".

You should place it at the end of the registerBundles() method, between

    "$bundles = array_merge($bundles, $pimBundles);"

and

    "return $bundles;".

Then clear the cache and restart apache again. The connector should appear in the proposed export profiles.
